images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -79,13 +79,14 @@ foreach ($arch in $archs)
 Write-Host "`n$dash Clean packages"
 pacman.exe -Scc --noconfirm
 
+$pkgs = $(pacman.exe -Q)
 Write-Host "`n$dash Installed msys2 packages"
-pacman.exe -Q | grep -v ^mingw-w64-
+$pkgs | grep -v ^mingw-w64-
```

Comment:
Why should we output twice the same pkgs at 84 line and 89? `^mingw-w64 `  regex covers all archs:
```
$archs:
mingw-w64-x86_64
mingw-w64-i686
mingw-w64-ucrt-x86_64
```

Reply:
yep, that really duplicates output :) thanks!